src/Core/Checkout/Test/Customer/SalesChannel/RemoveWishlistProductRouteTest.php

```diff
@@ -172,6 +182,26 @@ public function testDeleteProductShouldThrowWishlistProductNotFoundException():
         static::assertEquals('Wishlist product with id ' . $productId . ' not found', $errors['detail']);
     }
 
+    public function testEventDispatchWishlistRemoveAddedEvent(): void
```

Comment:
Is here a difference to the stuff you test in `testDeleteProductShouldReturnSuccess`?

Sure you assert that `$event->getWishlistId()` returns the correct id, but you should be able to add that assertion to the existing testcase, IMHO there is no need for a specific test case

Reply:
ah you already fixed this one :+1: